pycfmodel/model/generic.py

```diff
@@ -18,21 +19,27 @@
     ResolvableStrOrList,
 )
 
-
-class _Auxiliar(BaseModel):
-    aux: Union[
+AuxType = Annotated[
+    Union[
         FunctionDict,
         Properties,
         ResolvableBoolOrList,
         ResolvableIntOrList,
-        ResolvableDatetimeOrList,
         ResolvableDateOrList,
+        ResolvableDatetimeOrList,
```

Comment:
Changes the order due to the new lax config of pydantic. Date can be parsed inside a Datetime

Reply:
It might be a good idea to add a comment mentioning this as it won't be obvious going forward